src/natcap/invest/utils.py

```diff
@@ -582,8 +582,13 @@ def read_csv_to_dataframe(
     # allow encoding kwarg to override this if it's provided
     if not encoding and has_utf8_bom(path):
         encoding = 'utf-8-sig'
-    dataframe = pandas.read_csv(path, engine=engine, encoding=encoding,
-                                sep=sep, **kwargs)
+    try:
+        dataframe = pandas.read_csv(path, engine=engine, encoding=encoding,
+                                    sep=sep, **kwargs)
+    except UnicodeDecodeError as error:
+        LOGGER.error(
+            f'{path} must be encoded as utf-8 or avoid non-ASCII characters')
```

Comment:
This is super minor, but we could simplify the sentence by avoiding the double-negative: "must be encoded as utf-8 or ASCII"

Reply:
That is better, thanks!